src/vm/tieredcompilation.cpp

```diff
@@ -718,6 +719,8 @@ void TieredCompilationManager::ActivateCodeVersion(NativeCodeVersion nativeCodeV
     // code version will activate then.
     ILCodeVersion ilParent;
     HRESULT hr = S_OK;
+    MethodDescBackpatchInfoTracker::ConditionalLockHolder lockHolder;
```

Comment:
This could also pass in `pMethod->MayHaveEntryPointSlotsToBackpatch()` as a parameter